backend/laundry/management/commands/update_laundry_rooms.py

```diff
@@ -0,0 +1,66 @@
+import csv
+from functools import reduce
+
+import requests
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from requests.exceptions import HTTPError
+
+
+class Command(BaseCommand):
+    help = "Update laundry rooms csv from server"
+
+    def handle(self, *args, **kwargs):
+        # Pull initial request with everything
+        try:
+            headers = {
+                "x-api-key": settings.LAUNDRY_X_API_KEY,
+                "alliancels-auth-token": settings.LAUNDRY_ALLIANCELS_API_KEY,
+            }
+            all_rooms_request = requests.get(
+                f"{settings.LAUNDRY_URL}/geoBoundaries/5610?raw=true", timeout=60, headers=headers
+            )
+            all_rooms_request.raise_for_status()
+
+        except HTTPError as e:
+            self.stdout.write(f"Error: {e}")
+            return
+
+        all_rooms_request_json = all_rooms_request.json()
+        locations = all_rooms_request_json["geoBoundaries"][0]["geoBoundaries"]
+
+        laundry_rooms = [
+            [room["id"], room["roomName"], location["description"], location["id"]]
```

Comment:
use a dictionary, not a list of lists, like
```
{
  "room_id": ...
   ...
}
```

Reply:
ifl u could also just put the room object itself

Reply:
but yea lets not have indexes be symbolic lets be explicit